---
layout: page
title: "Publications"
description: "Things I've authored"
header-img: "../img/bg.jpg"
order: 1
---

Metrics: &nbsp;*[Google Scholar](https://scholar.google.com/citations?user=F3yZjVMAAAAJ)*
<br>

# Selected
1.  Maragkakis M, et al., *DIANA-microT web server: elucidating miRNA
functions through target prediction.* **Nucleic Acids Res.** 37, W273–6,
(2009).
[more...](http://nar.oxfordjournals.org/content/37/suppl_2/W273.abstract)<br/>
*(500+ citations)*

1.  Maragkakis M, et al., *Accurate miRNA target prediction correlates with
protein repression levels.* **BMC Bioinformatics** 10, 295 (2009).
[more...](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-295)<br/>
*(340+ citations)*

1.  Maragkakis M\*, Vergoulis T\*, et al., *DIANA-microT Web server upgrade
supports Fly and Worm miRNA target prediction and bibliographic miRNA to
disease association.* **Nucleic Acids Res.** 39, W145–8 (2011).
[more...](http://nar.oxfordjournals.org/content/39/suppl_2/W145.abstract)<br/>
*(100+ citations)*

1.  Reczko M\*, Maragkakis M\*, et al., *Functional microRNA targets in
protein coding sequences.* **Bioinformatics** 28, 771–776 (2012).
[more...](http://bioinformatics.oxfordjournals.org/content/28/6/771.abstract)<br/>
*(240+ citations)*

1.  Vourekas A\*, Zheng K\*, Fu Q\*, Maragkakis M\*, et al., *The RNA helicase
MOV10L1 binds piRNA precursors to initiate piRNA processing.* **Genes &
Development** 29, 617–629 (2015).
[more...](http://genesdev.cshlp.org/content/early/2015/03/10/gad.254631.114)

1.  Maragkakis M\*, Alexiou P\*, Nakaya T, Mourelatos Z, *CLIPSeqTools - A
novel bioinformatics CLIP-Seq analysis suite.* **RNA** 22, 1–9 (2016).
[more...](http://rnajournal.cshlp.org/content/early/2015/11/17/rna.052167.115)

1.  Vourekas A, ALexiou P, Vrettos N, Maragkakis M, Mourelatos Z,
*Sequence-dependent but not sequence-specific piRNA adhesion traps mRNAs to
the germ plasm.* **Nature** 531, 390–394 (2016).
[more...](http://www.nature.com/nature/journal/v531/n7594/full/nature17150.html)<br/>
*(Highlighted in: Dev Cell. 37:9-10, 2016)*

1. Ibrahim F\*, Maragkakis M\*, Alexiou P, Mourelatos Z, *Ribothrypsis, a novel
process of canonical mRNA decay, mediates ribosome-phased mRNA
endonucleolysis.* **Nat. Struct. Mol. Biol.**, (2018).
[more...](http://rdcu.be/IC30)<br/>
*(selected for journal cover and research highlight)*

# Publications
1.  Maragkakis M, Reczko M, Simossis VA, Alexiou P, Papadopoulos GL, Dalamagas
T, Giannopoulos G, Goumas G, Koukis E, Kourtis K, Vergoulis T, Koziris N,
Sellis T, Tsanakas P, Hatzigeorgiou AG, *DIANA-microT web server: elucidating
miRNA functions through target prediction.* **Nucleic Acids Res.** 37,
W273–6, (2009).
[more...](http://nar.oxfordjournals.org/content/37/suppl_2/W273.abstract)

1.  Papadopoulos GL, Alexiou P, Maragkakis M, Reczko M, Hatzigeorgiou AG,
*DIANA-mirPath: Integrating human and mouse miRNAs in pathways.*
**Bioinformatics.** 25, 1991–1993 (2009).
[more...](http://bioinformatics.oxfordjournals.org/content/25/15/1991.abstract)

1.  Maragkakis M, Alexiou P, Papadopoulos GL, Reczko M, Dalamagas T,
Giannopoulos G, Goumas G, Koukis E, Kourtis K, Simossis VA, Sethupathy P,
Vergoulis T, Koziris N, Sellis T, Tsanakas P, Hatzigeorgiou AG, *Accurate
miRNA target prediction correlates with protein repression levels.* **BMC
Bioinformatics** 10, 295 (2009).
[more...](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-295)

1.  Alexiou P, Maragkakis M, Papadopoulos GL, Reczko M, Hatzigeorgiou AG,
*Lost in translation: an assessment and perspective for computational miRNA
target identification.* **Bioinformatics** 25, 3049–3055 (2009).
[more...](http://bioinformatics.oxfordjournals.org/content/25/23/3049.abstract)

1.  Alexiou P, Maragkakis M, Papadopoulos GL, Simmosis VA, Zhang L,
Hatzigeorgiou AG, *The DIANA-mirExTra web server: from gene expression data
to miRNA function.* **PLoS One** 5, e9171 (2010).
[more...](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009171)

1.  Iizasa H, Wulff BE, Alla NR, Maragkakis M, Megraw M, Hatzigeorgiou A,
Iwakiri D, Takada K, Wiedmer A, Showe L, Lieberman P, Nishikura K, *Editing
of Epstein-Barr virus-encoded BART6 miRNAs controls their dicer targeting and
consequently affects viral latency.* **J. Biol Chem.** 285, 33358–33370 (2010).
[more...](http://www.jbc.org/content/285/43/33358.abstract)

1.  Alexiou P, Maragkakis M, Hatzigeorgiou AG, *Online resources for miRNA
analysis.* **J. Nucleic Acids Investig.** 2, 2–5 (2011).
[more...](http://www.pagepress.org/journals/index.php/jnai/article/view/jnai.2011.e4)

1.  Maragkakis M\*, Vergoulis T\*, Alexiou P, Reczko M, Plomaritou K, Gousis M,
Kourtis K, Koziris N, Dalamagas T, Hatzigeorgiou AG, *DIANA-microT Web server
upgrade supports Fly and Worm miRNA target prediction and bibliographic miRNA
to disease association.* **Nucleic Acids Res.** 39, W145–8 (2011).
[more...](http://nar.oxfordjournals.org/content/39/suppl_2/W145.abstract)

1.  Reczko M\*, Maragkakis M\*, Alexiou P, Papadopoulos GL, Hatzigeorgiou AG,
*Accurate microRNA target prediction using detailed binding site accessibility
and machine learning on proteomics data.* **Front. Genet.** 2, 103 (2011).
[more...](http://journal.frontiersin.org/article/10.3389/fgene.2011.00103/abstract)

1.  Vergoulis T, Vlachos IS, Alexiou P, Georgakilas G, Maragkakis M, Reczko M,
Gerangelos S, Koziris N, Dalamagas T, Hatzigeorgiou AG, *TarBase 6.0:
capturing the exponential growth of miRNA targets with experimental support.*
**Nucleic Acids Res.** 40, D222–9 (2012).
[more...](http://nar.oxfordjournals.org/content/40/D1/D222.abstract)

1.  Reczko M\*, Maragkakis M\*, Alexiou P, Grosse I, Hatzigeorgiou AG,
*Functional microRNA targets in protein coding sequences.* **Bioinformatics**
28, 771–776 (2012).
[more...](http://bioinformatics.oxfordjournals.org/content/28/6/771.abstract)

1.  Vlachos IS, Kostoulas N, Vergoulis T, Georgakilas G, Reczko M, Maragkakis
M, Paraskevopoulou MD, Prionidis K, Dalamagas T, Hatzigeorgiou AG, *DIANA
miRPath v. 2.0: investigating the combinatorial effect of microRNAs in
pathways.* **Nucleic Acids Res.** 40, W498–504 (2012).
[more...](http://nar.oxfordjournals.org/content/40/W1/W498.abstract)

1.  Vourekas A, Zheng Q, Alexiou P\*, Maragkakis M\*, Kirino Y, Gregory BD,
Mourelatos Z, *Target RNA repertoire of Mili and Miwi reveals piRNA biogenesis
and function of Miwi in spermiogenesis.* **Nat. Struct. Mol. Biol.** 19,
773–781 (2012).
[more...](http://www.nature.com/nsmb/journal/v19/n8/full/nsmb.2347.html)

1.  Paraskevopoulou MD, Georgakilas G, Kostoulas N, Reczko M, Maragkakis M,
Dalamagas TM, Hatzigeorgiou AG, *DIANA-LncBase: experimentally verified and
computationally predicted microRNA targets on long non-coding RNAs.* **Nucleic
Acids Res.** 41, D239–45 (2013).
[more...](http://nar.oxfordjournals.org/content/41/D1/D239.abstract)

1.  Ibrahim F, Maragkakis M\*, Alexiou P\*, Maronski AM, Dichter AM,
Mourelatos Z, *Identification of in vivo, conserved, TAF15 RNA binding sites
reveals the impact of TAF15 on the neuronal transcriptome.* **Cell Reports**
3, 301–308 (2013).
[more...](http://www.cell.com/cell-reports/abstract/S2211-1247(13)00028-4)

1.  Nakaya T, Alexiou P\*, Maragkakis M\*, Chang A, Mourelatos Z, *FUS regulates
genes coding for RNA binding proteins in neurons by binding to their highly
conserved introns.* **RNA** 19, 498–509 (2013).
[more...](http://rnajournal.cshlp.org/content/19/4/498.abstract)

1.  Honda S, Kirino Y, Maragkakis M, Alexiou P, Ohtaki A, Murali R, Mourelatos
Z, Kirino Y, *Mitochondrial protein BmPAPI modulates the length of mature
piRNAs.* **RNA** 19, 1405–1418 (2013).
[more...](http://rnajournal.cshlp.org/content/19/10/1405.abstract)

1.  Liu X, Zheng Q, Vrettos N, Maragkakis M, Alexiou P, Gregory BD, Mourelatos
Z, *A MicroRNA precursor surveillance system in quality control of MicroRNA
synthesis.* **Molecular Cell** 55, 868–879 (2014).
[more...](http://www.cell.com/molecular-cell/abstract/S1097-2765(14)00610-8)

1.  Vourekas A\*, Zheng K\*, Fu Q\*, Maragkakis M\*, Alexiou P, Pillai RS,
Mourelatos Z, Wang PJ, *The RNA helicase MOV10L1 binds piRNA precursors to
initiate piRNA processing.* **Genes & Development** 29, 617–629 (2015).
[more...](http://genesdev.cshlp.org/content/early/2015/03/10/gad.254631.114)

1.  Maragkakis M\*, Alexiou P\*, Nakaya T, Mourelatos Z, *CLIPSeqTools - A novel
bioinformatics CLIP-Seq analysis suite.* **RNA** 22, 1–9 (2016).
[more...](http://rnajournal.cshlp.org/content/early/2015/11/17/rna.052167.115)

1.  Vourekas A, ALexiou P, Vrettos N, Maragkakis M, Mourelatos Z,
*Sequence-dependent but not sequence-specific piRNA adhesion traps mRNAs to
the germ plasm.* **Nature** 531, 390–394 (2016).
[more...](http://www.nature.com/nature/journal/v531/n7594/full/nature17150.html)

1. Vrettos N\*, Maragkakis M\*, Alexiou P, Mourelatos Z, *Kc167, a widely used
Drosophila cell line, contains an active primary piRNA pathway.* **RNA** 23, 108–118 (2016).
[more...](http://rnajournal.cshlp.org/content/early/2016/10/27/rna.059139.116.abstract)

1. Kortschak RD, Snyder JB, Maragkakis M, Adelson DL, *bíogo: a simple
high-performance bioinformatics toolkit for the Go language.* **JOSS**,
(2017).
[more...](http://joss.theoj.org/papers/10.21105/joss.00167)

1. Ibrahim F\*, Maragkakis M\*, Alexiou P, Mourelatos Z, *Ribothrypsis, a
novel process of canonical mRNA decay, mediates ribosome-phased mRNA
endonucleolysis.* **Nat. Struct. Mol. Biol.**, (2018).
[more...](http://rdcu.be/IC30)

# Book chapters and preprints
1.  Maragkakis M, Alexiou P, Mourelatos Z, *GenOO: A Modern Perl Framework
for High Throughput Sequencing analysis.* BioRxiv, (2015).
[more...](http://biorxiv.org/content/early/2015/11/03/019265)

1. Alexiou P, Maragkakis M, Mourelatos Z, Vourekas A, cCLIP-Seq: Retrieval of
Chimeric Reads from HITS-CLIP (CLIP-Seq) Libraries. In: Okamura K., Nakanishi
K. (eds) Argonaute Proteins. Methods in Molecular Biology, vol 1680. Humana
Press, New York, NY (2018).
[more...](https://dx.doi.org/10.1007/978-1-4939-7339-2_6)
