---
layout: page
title: "Publications"
description: "Things I've authored"
header-img: "../img/bg.jpg"
---

# Citation Metrics

| citations: | &nbsp;**2319** |
| h-index:	 | &nbsp;**15**   |
| i10-index: | &nbsp;**17**   |

[Google Scholar](https://scholar.google.com/citations?user=F3yZjVMAAAAJ&hl=en)  
[ResearcherID](http://www.researcherid.com/rid/F-5427-2010)  
[ResearchGate](https://www.researchgate.net/profile/Manolis_Maragkakis)


# Selected Publications

*Sequence-dependent but not sequence-specific piRNA adhesion traps mRNAs to
the germ plasm*, **Nature**, 2016.
([link](http://www.nature.com/nature/journal/v531/n7594/full/nature17150.html))

*The RNA helicase MOV10L1 binds piRNA precursors to initiate piRNA
processing.*, **Genes Dev**, 2015.
([link](http://genesdev.cshlp.org/content/early/2015/03/10/gad.254631.114))

*A MicroRNA precursor surveillance system in quality control of MicroRNA
synthesis.*, **Mol Cell**, 2014.
([link](http://www.cell.com/molecular-cell/abstract/S1097-2765(14)00610-8))

*FUS regulates genes coding for RNA binding proteins in neurons by binding
to their highly conserved introns.*, **RNA**, 2013.
([link](http://rnajournal.cshlp.org/content/19/4/498.abstract))

*Identification of in vivo, conserved, TAF15 RNA binding sites reveals the
impact of TAF15 on the neuronal transcriptome.*, **Cell Reports**, 2013.
([link](http://www.cell.com/cell-reports/abstract/S2211-1247(13)00028-4))

*Target RNA repertoire of Mili and Miwi reveals piRNA biogenesis and
function of Miwi in spermiogenesis.*, **Nat Struct Mol Biol**, 2012.
([link](http://www.nature.com/nsmb/journal/v19/n8/full/nsmb.2347.html))

# Peer-reviewed Journal Articles

1.  Vourekas A, ALexiou P, Vrettos N, Maragkakis M, Mourelatos Z,
*Sequence-dependent but not sequence-specific piRNA adhesion traps mRNAs to
the germ plasm*, **Nature**, 2016.
([link](http://www.nature.com/nature/journal/v531/n7594/full/nature17150.html))

1.  Maragkakis M\*, Alexiou P\*, Nakaya T, Mourelatos Z, *CLIPSeqTools - A novel
bioinformatics CLIP-Seq analysis suite*, **RNA**, 2015.
([link](http://rnajournal.cshlp.org/content/early/2015/11/17/rna.052167.115))

1.  Vourekas A\*, Zheng K\*, Fu Q\*, Maragkakis M\*, Alexiou P, Pillai RS,
Mourelatos Z, Wang PJ, *The RNA helicase MOV10L1 binds piRNA precursors to
initiate piRNA processing.*, **Genes Dev**, 2015.
([link](http://genesdev.cshlp.org/content/early/2015/03/10/gad.254631.114))

1.  Liu X, Zheng Q, Vrettos N, Maragkakis M, Alexiou P, Gregory BD, Mourelatos
Z, *A MicroRNA precursor surveillance system in quality control of MicroRNA
synthesis.*, **Mol Cell**, 2014.
([link](http://www.cell.com/molecular-cell/abstract/S1097-2765(14)00610-8))

1.  Honda S, Kirino Y, Maragkakis M, Alexiou P, Ohtaki A, Murali R, Mourelatos
Z, Kirino Y, *Mitochondrial protein BmPAPI modulates the length of mature
piRNAs.*, **RNA**, 2013.
([link](http://rnajournal.cshlp.org/content/19/10/1405.abstract))

1.  Nakaya T, Alexiou P\*, Maragkakis M\*, Chang A, Mourelatos Z, *FUS regulates
genes coding for RNA binding proteins in neurons by binding to their highly
conserved introns.*, **RNA**, 2013.
([link](http://rnajournal.cshlp.org/content/19/4/498.abstract))

1.  Ibrahim F, Maragkakis M\*, Alexiou P\*, Maronski AM, Dichter AM, Mourelatos Z,
*Identification of in vivo, conserved, TAF15 RNA binding sites reveals the
impact of TAF15 on the neuronal transcriptome.*, **Cell Reports**, 2013.
([link](http://www.cell.com/cell-reports/abstract/S2211-1247(13)00028-4))

1.  Paraskevopoulou MD, Georgakilas G, Kostoulas N, Reczko M, Maragkakis M,
Dalamagas TM, Hatzigeorgiou AG, *DIANA-LncBase: experimentally verified and
computationally predicted microRNA targets on long non-coding RNAs.*,
**Nucleic Acids Res.**, 2013.
([link](http://nar.oxfordjournals.org/content/41/D1/D239.abstract))

1.  Vourekas A, Zheng Q, Alexiou P\*, Maragkakis M\*, Kirino Y, Gregory BD,
Mourelatos Z, *Target RNA repertoire of Mili and Miwi reveals piRNA
biogenesis and function of Miwi in spermiogenesis.*, **Nat Struct Mol Biol**,
2012.
([link](http://www.nature.com/nsmb/journal/v19/n8/full/nsmb.2347.html))

1.  Vlachos IS, Kostoulas N, Vergoulis T, Georgakilas G, Reczko M, Maragkakis
M, Paraskevopoulou MD, Prionidis K, Dalamagas T, Hatzigeorgiou AG, *DIANA
miRPath v. 2.0: investigating the combinatorial effect of microRNAs in
pathways.*, **Nucleic Acids Res.**, 2012.
([link](http://nar.oxfordjournals.org/content/40/W1/W498.abstract))

1.  Reczko M\*, Maragkakis M\*, Alexiou P, Grosse I, Hatzigeorgiou AG, *Functional
microRNA targets in protein coding sequences.*, **Bioinformatics**, 2012.
([link](http://bioinformatics.oxfordjournals.org/content/28/6/771.abstract))

1.  Vergoulis T, Vlachos IS, Alexiou P, Georgakilas G, Maragkakis M, Reczko M,
Gerangelos S, Koziris N, Dalamagas T, Hatzigeorgiou AG, *TarBase 6.0:
capturing the exponential growth of miRNA targets with experimental support.*,
**Nucleic Acids Res.**, 2012.
([link](http://nar.oxfordjournals.org/content/40/D1/D222.abstract))

1.  Reczko M\*, Maragkakis M\*, Alexiou P, Papadopoulos GL, Hatzigeorgiou AG,
*Accurate microRNA target prediction using detailed binding site accessibility
and machine learning on proteomics data.*, **Frontiers in genetics**, 2011.
([link](http://journal.frontiersin.org/article/10.3389/fgene.2011.00103/abstract))

1.  Maragkakis M\*, Vergoulis T\*, Alexiou P, Reczko M, Plomaritou K, Gousis M,
Kourtis K, Koziris N, Dalamagas T, Hatzigeorgiou AG, *DIANA-microT Web server
upgrade supports Fly and Worm miRNA target prediction and bibliographic miRNA
to disease association.*, **Nucleic Acids Res.**, 2011.
([link](http://nar.oxfordjournals.org/content/39/suppl_2/W145.abstract))

1.  Alexiou P, Maragkakis M, Hatzigeorgiou AG, *Online resources for miRNA
analysis.*, **Journal of Nucleic Acids Investigation**, 2011.
([link](http://www.pagepress.org/journals/index.php/jnai/article/view/jnai.2011.e4))

1.  Iizasa H, Wulff BE, Alla NR, Maragkakis M, Megraw M, Hatzigeorgiou A,
Iwakiri D, Takada K, Wiedmer A, Showe L, Lieberman P, Nishikura K, *Editing
of Epstein-Barr virus-encoded BART6 miRNAs controls their dicer targeting and
consequently affects viral latency.*, **J Biol Chem.**, 2010.
([link](http://www.jbc.org/content/285/43/33358.abstract))

1.  Alexiou P, Maragkakis M, Papadopoulos GL, Simmosis VA, Zhang L,
Hatzigeorgiou AG, *The DIANA-mirExTra web server: from gene expression data
to miRNA function.*, **PLoS One**, 2010.
([link](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009171))

1.  Alexiou P, Maragkakis M, Papadopoulos GL, Reczko M, Hatzigeorgiou AG,
*Lost in translation: an assessment and perspective for computational miRNA
target identification.*, **Bioinformatics**, Review, 2009.
([link](http://bioinformatics.oxfordjournals.org/content/25/23/3049.abstract))

1.  Maragkakis M, Alexiou P, Papadopoulos GL, Reczko M, Dalamagas T,
Giannopoulos G, Goumas G, Koukis E, Kourtis K, Simossis VA, Sethupathy P,
Vergoulis T, Koziris N, Sellis T, Tsanakas P, Hatzigeorgiou AG, *Accurate
miRNA target prediction correlates with protein repression levels.*, **BMC
Bioinformatics**, 2009.
([link](http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-295))

1.  Papadopoulos GL, Alexiou P, Maragkakis M, Reczko M, Hatzigeorgiou AG,
*DIANA-mirPath: Integrating human and mouse miRNAs in pathways.*,
**Bioinformatics.**, 2009.
([link](http://bioinformatics.oxfordjournals.org/content/25/15/1991.abstract))

1.  Maragkakis M, Reczko M, Simossis VA, Alexiou P, Papadopoulos GL, Dalamagas
T, Giannopoulos G, Goumas G, Koukis E, Kourtis K, Vergoulis T, Koziris N,
Sellis T, Tsanakas P, Hatzigeorgiou AG, *DIANA-microT web server: elucidating
miRNA functions through target prediction.*, **Nucleic Acids Res.**, 2009.
([link](http://nar.oxfordjournals.org/content/37/suppl_2/W273.abstract))

# Conference Articles and Posters
1.  Maragkakis M, Alexiou P, Tadashi N, Mourelatos Z, *CLIPSeqTools: A novel
bioinformatics CLIP-Seq analysis suite applied on Human Brain Ago HITS-CLIP.*,
**HSCBB14**, 2014

1.  Maragkakis M, Alexiou P, Mourelatos Z, *GenOO-HTS: A Modern Perl Framework
for High Throughput Sequencing analysis.*, **HiTSEQ 2013: High Throughput
Sequencing Algorithms & Applications**, 2013

1. Vergoulis T, Alexakis M, Dalamagas T, Maragkakis M, Hatzigeorgiou AG,
Sellis T, *Tarcloud: A cloud-based platform to support mirna target
prediction.*, **Lecture Notes in Computer Science**, 2012.
([link](http://link.springer.com/chapter/10.1007%2F978-3-642-31235-9_48))

1.  Grau J, Arend D, Grosse I, Hatzigeorgiou AG, Keilwagen J, Maragkakis M,
Weinholdt C, Posch S, *Predicting miRNA targets utilizing an extended profile
HMM.*, **25th German Conference on Bioinformatics**, 2010.
([link](https://www.researchgate.net/publication/221493448_Predicting_miRNA_targets_utilizing_an_Extended_Profile_HMM))

1.  Dalamagas, T, Farmakakis, T, Maragkakis, M, Hatzigeorgiou, AG, *FreePub:
Collecting and Organizing Scientific Material Using Mindmaps.*, **3rd
International Workshop on Semantic Web Applications and Tools for the Life
Sciences**, 2010.
([link](http://ceur-ws.org/Vol-698/paper11.pdf))

# Non Peer-reviewed Articles

1.  Maragkakis M, Alexiou P, Mourelatos Z, *GenOO: A Modern Perl Framework
for High Throughput Sequencing analysis.*, **BioRxiv**, 2015.
([link](http://biorxiv.org/content/early/2015/11/03/019265))
